ksuid_test.go

```diff
@@ -266,6 +266,46 @@ func TestSort(t *testing.T) {
 	}
 }
 
+func TestNext(t *testing.T) {
+	id0 := Nil
```

Comment:
I think these could be  better written as subtests named `min`/`max` rather than re-use the `id..` variables.

Reply:
That's a good idea 👍